wicker/core/datasets.py

```diff
@@ -125,79 +135,134 @@ def __len__(self) -> int:
         pass
 
```

Comment:
Please pardon, I must be super OCD and re-order the `arrow_table` function to come before `schema` function so that they are in alphabetic order. Maybe I need to go to coding mental hospital.

Reply:
Cool， I am good with reformat at once